README.md

```diff
@@ -44,6 +44,7 @@ Please check the [contribution guidelines](CONTRIBUTING.md) for info on formatti
 - [EthereumDev.io](https://ethereumdev.io) - Complete tutorial from starting to writing complex smart contracts and DApps.
 - [Learn Solidity](https://github.com/willitscale/learning-solidity) - Complete guide on getting started, creating your own crypto, ICOs and deployment.
 - [CryptoZombies](https://cryptozombies.io) - Interactive code school that teaches you to write smart contracts through building your own crypto-collectables game.
+- [Community Curated Solidity Resources](https://hackr.io/tutorials/learn-solidity)
```

Comment:
Thanks for your contribution @qualityjacks!

Two more things needed here:

Reply:
Thanks, submitted another request.

Reply:
Thanks! 👍  Btw it's cleaner if you just push another commit to an existing PR and the maintainers can squash it into one commit for you on merge.